### ISSUE TITLE
 TrackPlayer.STATE_PLAYING being null

### ISSUE THREAD

**Issue by SatoshiInoue** [react-native-kit/react-native-track-player#141]
I think I found a bug - in iOS, TrackPlayer.STATE_PLAYING isn't exported into the module and it's being a null. The version is 0.2.2.

```
let state = await TrackPlayer.getState();
console.log('TrackPlayer', TrackPlayer.STATE_PLAYING); // This is 'null
if (state == TrackPlayer.STATE_PLAYING) {/*<= this is always false*/}        
if (state == "STATE_PLAYING") {/*This works*/}
```

This doesn't happen in Android, only in iOS. Can this be fixed in the future release? This is not urgent at all b/c the workaround in the snippet does its job fine.

**Comment by SatoshiInoue** [comment]
This is actually a better workaround:
`if (state === "STATE_PLAYING" || state == 3) {}`
In Android, TrackPlayer.getState() returns an integer whereas in iOS it's a string.

**Comment by RassaLibre** [comment]
Do you have numbers for other states? Btw. this solution works like charm!

**Comment by SatoshiInoue** [comment]
Here's the list for Android: 
https://developer.android.com/reference/android/support/v4/media/session/PlaybackStateCompat.html
Check TrackModules.java in the package. The constants are from PlaybackStateCompat.

**Comment by ferrannp** [comment]
Found the same bug. On Android, `TrackPlayer.STATE_PLAYING` is `"STATE_PLAYING"`, on iPhone, it is `undefined`.

**Comment by callmetwan** [comment]
Here's a more complete snippet with custom constants defined so you don't have to go look them up:

```
import Platform from 'react-native'

export default RnTrackPlayerConstants = {
	STATE_NONE: Platform.OS === 'android' ? 0 : 'STATE_NONE',
	STATE_STOPPED: Platform.OS === 'android' ? 1 : 'STATE_NONE',
	STATE_PAUSED: Platform.OS === 'android' ? 2 : 'STATE_PAUSED',
	STATE_PLAY: Platform.OS === 'android' ? 3 : 'STATE_PLAY',
	STATE_BUFFERING: Platform.OS === 'android' ? 6 : 'STATE_BUFFERING',
}
```

**Comment by padupuy** [comment]
@SatoshiInoue
your problem seems to be resolved with the latest version **0.2.3**

I confirm that with version 0.2.2, all constants from TrackPlayer on iOS was empty, no problem with Android

<img width="1025" alt="capture d ecran 2018-04-05 a 10 28 59" src="https://user-images.githubusercontent.com/715255/38355180-68205ffc-38bc-11e8-957a-378c0512129e.png">

**Comment by stachu2k** [comment]
@padupuy Finally! This inconsistency gave me a lot of bugs when I started to work with this package. Documentation didn't include the differences between Android and iOS.
